includes/class-fs-api.php

```diff
@@ -318,9 +318,12 @@ function get( $path = '/', $flush = false, $expiration = WP_FS__TIME_24_HOURS_IN
 				$flush = true;
 			}
 
-			$cached_result = self::$_cache->get( $cache_key );
+            $has_valid_cache = self::$_cache->has_valid( $cache_key, $expiration );

```

Comment:
@fajardoleo please use `tab` for indentation. The rest of the file is using tab.